packages/hyperion-autologging/src/ALElementValuePublisher.ts

```diff
@@ -132,12 +136,12 @@ export function publish(options: InitOptions): void {
              * We know we are lookging for checkbox/radio inputs, so checked is good enough
              * Also, the following check ensures we are not reading PII from other inputs, in case we expand the search
              */
-            if (input.checked) {
+            if (includeInitialDisabledState || input.checked) {
```

Comment:
`includeInitialDisabledState` would be set by app config, right? Wondering if it's better to just remove this `if` check

Reply:
how would it work if the `includeInitialDisabledState` is false in the config?